app/models/planet.py

```diff
@@ -0,0 +1,31 @@
+from app import db
+from flask import abort, make_response
+
+class Planet(db.Model):
+    id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    name = db.Column(db.String)
+    description = db.Column(db.String)
+    circumference = db.Column(db.Integer)
+    length_of_year= db.Column(db.Integer)
```

Comment:
Should we be able to make planets with no name? To prevent our API from creating a planet with a `null` name, we can set that column to `nullable = False`.

```python
class Planet(db.Model):
    id = db.Column(db.Integer, primary_key=True, autoincrement=True)
    name = db.Column(db.String, nullable=False)
    description = db.Column(db.String)
    circumference = db.Column(db.Integer)
    length_of_year= db.Column(db.Integer)
```